src/Microsoft.OData.Core/ODataConstants.cs

```diff
@@ -177,16 +177,28 @@ internal static class ODataInternalConstants
         internal const string ContextUriFragmentUntyped = "Edm.Untyped";
 
         /// <summary>The $delta token indicates delta resource set.</summary>
-        internal const string ContextUriDeltaResourceSet = "/$delta";
+        internal const string DeltaResourceSet = "$delta";
+
+        /// <summary>The $delta token indicates delta resource set.</summary>
+        internal const string ContextUriDeltaResourceSet = UriSegmentSeparator + DeltaResourceSet;
 
         /// <summary>The $deletedEntity token indicates delta resource.</summary>
-        internal const string ContextUriDeletedEntry = "/$deletedEntity";
+        internal const string DeletedEntry = "$deletedEntity";
+
+        /// <summary>The $delta token indicates delta resource set.</summary>
+        internal const string ContextUriDeletedEntry = UriSegmentSeparator + DeletedEntry;
 
         /// <summary>The $delta token indicates delta link.</summary>

```

Comment:
>$delta [](start = 25, length = 6)

$link?
same for L194 below.

Reply:
yep. thanks.

---
In reply to: [150155329](https://github.com/OData/odata.net/pull/989#discussion_r150155329) [](ancestors = 150155329)